src/app/docs/getting-started/angular-cli/angular-cli.component.html

```diff
@@ -14,6 +14,17 @@ <h1>Setup with Angular CLI</h1>
 
   Navigate to `localhost:4201` and start sandboxing components!
 </section>
+<section>
+  <p>
+    <a
+      routerLink="/docs/getting-started/angular-cli"
+      [fragment]="getFragment()">
+      Click here to {{ showingExtraContent ? 'hide' : 'show' }} the in-depth explanation of what's new!
```

Comment:
I think we should change this to be "Click here to show the in-depth explanation of the configuration!"